packages/react-components/src/components/MessageThread.tsx

```diff
@@ -323,7 +333,7 @@ const memoizeAllMessages = memoizeFnAll(
         const personaOptions: IPersona = {
           hidePersonalDetails: true,
           size: PersonaSize.size32,
-          text: message.senderDisplayName
+          imageInitials: getInitials(message.senderDisplayName, false, true)
```

Comment:
Didn't know they exported this, a great finding!

Reply:
"Another nice way to do it is just adding hidePersonaDetails (might save us tons of time of writing an initials function), with text still there, which will hide the name for you, check my codepen for an example:" - PorterNan

Issue is, the hidePersonalDetails is set to true, even in the react components when looking through devtools... I will investigate further as to why storybook continues to display the name even with hidePersonalDetails set to true.

Reply:
Found that when "text" prop gets populated, a TooltipHostBase gets generated as well. If I manually set the children to empty string "" through dev tools, the name disappears on top of the message box.

If "text" prop does not get populated, then no TooltipHostBase gets generated.

![Screen Shot 2022-01-26 at 2 01 24 PM](https://user-images.githubusercontent.com/97130533/151255252-5be16002-9d19-4603-a4c5-04a168a02955.png)

Reply:
Resolved. Noticed there was humanerror previously. Instead of the prop "hidePersonaDetails" being used, it was misspelt previously as "hidePersona**l**Details".